Dockerfile

```diff
@@ -1,14 +1,22 @@
 # Container for building Go binary.
 FROM golang:1.23.1-bookworm AS builder
+
 # Install dependencies
-RUN apt-get update && apt-get install -y build-essential git
+RUN apt-get update && apt-get install -y --no-install-recommends build-essential git \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
Not necessary, as this is multi-stage build and the image from the first stage is not used in the final image.

Reply:
I removed the cache deletion, but now the code quality test suggests that the cache must be cleared.